src/MeMoModel.py

```diff
@@ -81,11 +81,13 @@ def annotate(self) -> None:
         print("Total:", anno_result)
 
 
-    def match(self, model2: MeMoModel, keep1ToMany:bool = True) -> pd.DataFrame:
-        """ compares the metabolites of two models and returns a data frame with additional information """
+    def match(self, model2: MeMoModel, keep1ToMany:bool = True, output_names: bool = False) -> pd.DataFrame:
```

Comment:
IF https://github.com/Porthmeus/MeMoMe/pull/89 gets merged we have to add a parameter to the main program that enables/disables the output_names stuff

Reply:
#89 got merged. Add the additional parameter ASAP

Reply:
please fix #89 and I will review